seacrowd/config_helper.py

```diff
@@ -0,0 +1,1038 @@
+    'kopi_cc_2022_21-neardup_clean_seacrowd_ssp',
+    'kopi_cc_2022_27-raw_seacrowd_ssp',
+    'kopi_cc_2022_27-dedup_seacrowd_ssp',
+    'kopi_cc_2022_27-neardup_seacrowd_ssp',
+    'kopi_cc_2022_27-neardup_clean_seacrowd_ssp',
+    'kopi_cc_news_2016_source',
+    'kopi_cc_news_2017_source',
+    'kopi_cc_news_2018_source',
+    'kopi_cc_news_2019_source',
+    'kopi_cc_news_2020_source',
+    'kopi_cc_news_2021_source',
+    'kopi_cc_news_2022_source',
+    'kopi_cc_news_all_source',
+    'kopi_cc_news_2016_seacrowd_ssp',
+    'kopi_cc_news_2017_seacrowd_ssp',
+    'kopi_cc_news_2018_seacrowd_ssp',
+    'kopi_cc_news_2019_seacrowd_ssp',
+    'kopi_cc_news_2020_seacrowd_ssp',
+    'kopi_cc_news_2021_seacrowd_ssp',
+    'kopi_cc_news_2022_seacrowd_ssp',
+    'kopi_cc_news_all_seacrowd_ssp',
+    'kopi_nllb_all-raw_source',
+    'kopi_nllb_all-dedup_source',
+    'kopi_nllb_all-neardup_source',
+    'kopi_nllb_ace_Latn-raw_source',
+    'kopi_nllb_ace_Latn-dedup_source',
+    'kopi_nllb_ace_Latn-neardup_source',
+    'kopi_nllb_ban_Latn-raw_source',
+    'kopi_nllb_ban_Latn-dedup_source',
+    'kopi_nllb_ban_Latn-neardup_source',
+    'kopi_nllb_bjn_Latn-raw_source',
+    'kopi_nllb_bjn_Latn-dedup_source',
+    'kopi_nllb_bjn_Latn-neardup_source',
+    'kopi_nllb_ind_Latn-raw_source',
+    'kopi_nllb_ind_Latn-dedup_source',
+    'kopi_nllb_ind_Latn-neardup_source',
+    'kopi_nllb_jav_Latn-raw_source',
+    'kopi_nllb_jav_Latn-dedup_source',
+    'kopi_nllb_jav_Latn-neardup_source',
+    'kopi_nllb_min_Latn-raw_source',
+    'kopi_nllb_min_Latn-dedup_source',
+    'kopi_nllb_min_Latn-neardup_source',
+    'kopi_nllb_sun_Latn-raw_source',
+    'kopi_nllb_sun_Latn-dedup_source',
+    'kopi_nllb_sun_Latn-neardup_source',
+    'kopi_nllb_all-raw_seacrowd_ssp',
+    'kopi_nllb_all-dedup_seacrowd_ssp',
+    'kopi_nllb_all-neardup_seacrowd_ssp',
+    'kopi_nllb_ace_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_ace_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_ace_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_ban_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_ban_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_ban_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_bjn_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_bjn_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_bjn_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_ind_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_ind_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_ind_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_jav_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_jav_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_jav_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_min_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_min_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_min_Latn-neardup_seacrowd_ssp',
+    'kopi_nllb_sun_Latn-raw_seacrowd_ssp',
+    'kopi_nllb_sun_Latn-dedup_seacrowd_ssp',
+    'kopi_nllb_sun_Latn-neardup_seacrowd_ssp'
+]
+
+_RESOURCE_CONFIG_NAMES = [
+    'inset_lexicon_seacrowd_text',
+    'kamus_alay_seacrowd_t2t'
+]
+
+_CURRENTLY_BROKEN_NAMES = [
+
+]
+
+BENCHMARK_DICT = {
```

Comment:
will we add SEACrowd-based Benchmark to this later? or could it be done on this review since the dataset picking has been done

Reply:
@sabilmakbar Yes yes. I'll add it on the next PR since we're still reviewing some of the speech dataloaders.

Reply:
and for benchmark, we can use the datasets used for SEACrowd Experiment, hence the value should be overriden from NC